src/bin/rdcore/unique_fs.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::cmdline::*;
+use anyhow::{bail, Result};
+
+use libcoreinst::blockdev::*;
+
+pub fn verify_unique_fs(config: &VerifyUniqueFsLabelConfig) -> Result<()> {
+    let pts = get_filesystems_with_label(&config.label)?;
+    if pts.len() != 1 {
+        for pt in &pts {
+            eprintln!("- {}", pt);
```

Comment:
We're printing a list of devices first, without context, then failing with an error afterward.  It might make more sense to either print a header/trailer the way install does, or integrate the list into the `bail!()` call.